assets/xml/objects/object_dy_obj.xml

```diff
@@ -1,51 +1,62 @@
-        <Animation Name="object_dy_obj_Anim_002338" Offset="0x2338" /> <!-- Original name is "be_mahoukousi_loop" -->
-        <Animation Name="object_dy_obj_Anim_0027B0" Offset="0x27B0" /> <!-- Original name is "be_syutugen" -->
-        <Animation Name="object_dy_obj_Anim_0045FC" Offset="0x45FC" /> <!-- Original name is "be_utubuse" -->
-        <Animation Name="object_dy_obj_Anim_005238" Offset="0x5238" /> <!-- Original name is "be_utubuse_loop" -->
-        <Animation Name="object_dy_obj_Anim_005E20" Offset="0x5E20" /> <!-- Original name is "bigelf_denjyu" -->
-        <Animation Name="object_dy_obj_Anim_006DE4" Offset="0x6DE4" /> <!-- Original name is "bigelf_hakusyu" -->
-        <Animation Name="object_dy_obj_Anim_008090" Offset="0x8090" /> <!-- Original name is "bigelf_itemdasu" -->
-        <Animation Name="object_dy_obj_Anim_00C500" Offset="0xC500" /> <!-- Original name is "bigelf_syutugen" -->
-        <Animation Name="object_dy_obj_Anim_00D15C" Offset="0xD15C" /> <!-- Original name is "bigelf_udegumi" -->
-        <DList Name="object_dy_obj_DL_00D1B0" Offset="0xD1B0" />
-        <DList Name="object_dy_obj_DL_00D228" Offset="0xD228" />
-        <Texture Name="object_dy_obj_Tex_00D240" OutName="tex_00D240" Format="i4" Width="32" Height="32" Offset="0xD240" />
-        <Texture Name="object_dy_obj_Tex_00D440" OutName="tex_00D440" Format="i8" Width="16" Height="16" Offset="0xD440" />
-        <Texture Name="object_dy_obj_Tex_00D540" OutName="tex_00D540" Format="i8" Width="32" Height="64" Offset="0xD540" />
-        <Array Name="object_dy_obj_Vtx_00DD40" Count="27" Offset="0xDD40">
+        <!-- Great Fairy animations -->
+        <Animation Name="gGreatFairyStartGivingUpgradeAnim" Offset="0x129C" /> <!-- Original name is "be_mahoukousi" -->
+        <Animation Name="gGreatFairyGivingUpgradeAnim" Offset="0x2338" /> <!-- Original name is "be_mahoukousi_loop" -->
+        <Animation Name="gGreatFairyAnim_0027B0" Offset="0x27B0" /> <!-- Original name is "be_syutugen" --> <!-- unused -->
+        <Animation Name="gGreatFairyLayDownTransitionAnim" Offset="0x45FC" /> <!-- Original name is "be_utubuse" -->
+        <Animation Name="gGreatFairyLayingDownAnim" Offset="0x5238" /> <!-- Original name is "be_utubuse_loop" -->
+        <Animation Name="gGreatFairyTeachSpinAttackAnim" Offset="0x5E20" /> <!-- Original name is "bigelf_denjyu" ("instruction") -->
+        <Animation Name="gGreatFairyClappingAnim" Offset="0x6DE4" /> <!-- Original name is "bigelf_hakusyu" ("applause") -->
+        <Animation Name="gGreatFairyShowingItemAnim" Offset="0x8090" /> <!-- Original name is "bigelf_itemdasu" -->
+        <Animation Name="gGreatFairySpinLayDownAnim" Offset="0xC500" /> <!-- Original name is "bigelf_syutugen" -->
+        <Animation Name="gGreatFairyArmsFoldedAnim" Offset="0xD15C" /> <!-- Original name is "bigelf_udegumi" ("arms crossed") -->
```

Comment:
Would there be any objection to keeping it ArmsCrossed to be closer to the original?

Reply:
I would be careful in trusting my Japanese translations, because I don't actually know Japanese. In fact, looking at "udegumi" in a different dictionary gives this:
![image](https://github.com/zeldaret/mm/assets/12245827/5b3aadb8-0b93-41de-9540-e748d94ed559)